.buildkite/pipeline.yml

```
@@ -107,3 +107,69 @@ steps:
         depends_on:
           - "operator-image-build"
           - "e2e-tests-image-build"
+
+  - group: redhat-release
+
+    steps:
+
```

I would move the steps in a new pipeline `.buildkite/pipeline-release-operatorhub.yml` that we invoke from the root pipeline `.buildkite/pipeline.yml`.

```suggestion

      - label: "redhat operatorhub"
        if: build.message == "run operatorhub release"
        commands:
            - buildkite-agent pipeline upload .buildkite/pipeline-release-operatorhub.yml
```